randovania/layout/preset_migration.py

```diff
@@ -584,7 +584,10 @@ def _migrate_v25(preset: dict) -> dict:
 
     return preset
 
+
 def _migrate_v26(preset: dict) -> dict:
+    preset["configuration"]["minimum_available_locations_for_hint_placement"] = 0
+    preset["configuration"]["minimum_location_weight_for_hint_placement"] = 0.0
```

Comment:
this is bad form and you know it :P

Reply:
The reusing a preset version? I don't care, that was just merged lol

The bunch of similar fields? I could add a class but also bleh...

Reply:
my presets are ruined ;~;